florist/app/jobs/details/page.tsx

```diff
@@ -0,0 +1,242 @@
+"use client";
+
+import { useSearchParams } from "next/navigation";
+import Image from "next/image";
+
+import { ReactElement } from "react/React";
+
+import { useGetJob } from "../hooks";
+import { validStatuses, ClientInfo } from "../definitions";
+import loading_gif from "../../assets/img/loading.gif";
+
+export default function JobDetails(): ReactElement {
+    return (
+        <div className="col-12 align-items-center">
+            <JobDetailsHeader />
+            <JobDetailsBody />
+        </div>
+    );
+}
+
+export function JobDetailsHeader(): ReactElement {
+    return (
+        <div className="card-header pb-0 pt-3">
+            <h1 className="mt-3 mb-0">Job Details</h1>
+        </div>
+    );
+}
+
+export function JobDetailsBody(): ReactElement {
+    const searchParams = useSearchParams();
+    const jobId = searchParams.get("id");
+
+    const { data: job, error, isLoading } = useGetJob(jobId);
+
+    if (isLoading) {
+        return (
+            <div className="d-flex justify-content-center align-items-center vh-100">
+                <Image id="job-details-loading" src={loading_gif} alt="Loading" height={64} width={64} />
+            </div>
+        );
+    }
+
+    if (!job || error) {
+        if (error) {
+            console.error(error);
+        }
+        return (
+            <div className="container pt-3 p-0">
+                <div className="alert alert-danger text-white" id="job-details-error">
+                    Error retrieving job.
+                </div>
+            </div>
+        );
+    }
+
+    return (
+        <div className="container pt-3 p-0">
+            <div className="row">
+                <div className="col-sm-2">
+                    <strong className="text-dark">Job ID:</strong>
+                </div>
+                <div className="col-sm" id="job-details-id">
+                    {job._id}
+                </div>
+            </div>
+            <div className="row">
+                <div className="col-sm-2 align-content-center">
+                    <strong className="text-dark">Status:</strong>
+                </div>
+                <div className="col-sm">
+                    <JobDetailsStatus status={job.status} />
```

Comment:
small nit: The Status Button is a bit larger than the text so the spacing between the rows of server (and the server config below) are not uniform.  Maybe worth increasing size of the text or adding some padding perhaps if that looks slightly more visually pleasing

Reply:
You mean add padding on the other rows so the spacing is the same?

Reply:
Yeah! Like I said very small nit, if you don't think it really makes a difference feel free to disregard

Reply:
ok, will try :)

Reply:
Done, please check again.